apps/design/frontend/test/setupTests.ts

```diff
@@ -9,26 +10,23 @@ import {
 import { TextEncoder } from 'node:util';
 import { makeIdFactory } from './id_helpers';
 
-// styled-components version 5.3.1 and above requires this remapping for jest
-// environments, reference: https://github.com/styled-components/styled-components/issues/3570
-jest.mock('styled-components', () =>
-  jest.requireActual('styled-components/dist/styled-components.browser.cjs.js')
-);
+expect.extend(matchers);
 
 // Deterministic ID generation
 const idFactory = makeIdFactory();
 
 beforeEach(() => {
+  cleanup();
```

Comment:
Should this be run in `beforeEach` or `afterEach`? https://testing-library.com/docs/react-testing-library/api/#cleanup

If a test fails and this suite short-circuits will we violate the recommendation " If not, you will need to call cleanup() after each test."? I suppose if all suites call `cleanup()` in `beforEach` it doesn't matter.

Reply:
I'll switch it per their recommendation. However, I don't think it actually matters most of the time if ever.